### PR TITLE
i18n(backend)

### PR DESCRIPTION
**Pull request by joebordes**
Please review this carefully as I have no idea of clojure

### CODE REVIEW

File: src/metabase/driver.clj [hunk 1]
```diff
@@ -25,6 +25,9 @@
             [metabase.sync.interface :as si]
             [metabase.util :as u]
             [schema.core :as s]
+            [metabase.util.i18n :refer [available-locales-with-names set-locale]]
+            [metabase.util.password :as password]
```
Comment (senior) [inline src/metabase/driver.clj]: The above two namespaces shouldn't be needed on any of these files.

**Review by senior — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just need to remove those extra requires and then :+1:

**Comment by joebordes** [comment]
Ok, so I can just use "tru" freely in the clojure files, right?

Do you need help with those files?
Are there a lot of strings to do?

**Comment by joebordes** [comment]
I don't think I am going to be able to help much. I uploaded PR #7267 
I will try to find some time...

Let me know if there is anything precise you think I can help on.

**Review by senior — APPROVED** [review]

**Pull request merged**